fix for the Hyperledger explorer not working

ethicalvats opened the pull request:
fix: renamed the 'organizations' dir to 'crypto' in compose yml file and first-network config

opentaps commented:
@udosson What do you think of this change?

udosson commented:
I cannot reproduce the error. 
I took a look at the pull request and don't get why the changes should solve any connection issues. The proposed solution just changes paths inside the explorer docker container as well as in the connection-profile. It's a replacement of `organizations` with `crypto. Correct me, if I'm wrong.
@ethicalvats could you provide your error logs, please. 
Thanks a lot :)

ethicalvats commented:
> I cannot reproduce the error.
> I took a look at the pull request and don't get why the changes should solve any connection issues. The proposed solution just changes paths inside the explorer docker container as well as in the connection-profile. It's a replacement of `organizations` with `crypto. Correct me, if I'm wrong.
> @ethicalvats could you provide your error logs, please.
> Thanks a lot :)

@udosson I have got the explorer working. I was not specifically looking for the fix of the connection issue but what I did was, cloned the latest release of [explorer](https://github.com/hyperledger/blockchain-explorer) and tried connecting the carbonAccounting network to it, It was working. So my guess is, this project is using an old release of explorer (docker-compose set up) that doesn't work with latest version of HF. And indeed I found the [explorer setup](https://github.com/opentaps/blockchain-carbon-accounting/tree/master/utility-emissions-channel/docker-compose-setup/blockchain-explorer) was not updated from 2 months.

So my PR is using the latest supported config from explorer project. Here is the screenshot of the explorer running on my local machine.

<img width="1676" alt="Screenshot 2020-10-28 at 10 56 37 AM" src="https://user-images.githubusercontent.com/1000068/97396610-6bdf5600-190d-11eb-8d55-4039791c52f0.png">

Let me know if you need more info, thanks

ethicalvats commented:
@udosson another small change I did is to remove the command at the start of container

```
command: sh -c "node /opt/explorer/main.js && tail -f /dev/null"
```

They moved their startup process to a shell script, so it is no longer required to run the node script manually.

udosson commented:
@ethicalvats, thanks for clarification.
I updated the version of explorer and explorer db to the newest version (1.1.3) and removed the command at the start up of the container. Thanks for this hint.
Hyperledger Explorer is working without any issues with v1.1.3.
Thus I wouldn't merge your commit.
Are you okay with that?
It would be great if you could very that the latest commit is working at your machine.

Best
Robin

ethicalvats commented:
> @ethicalvats, thanks for clarification.
> I updated the version of explorer and explorer db to the newest version (1.1.3) and removed the command at the start up of the container. Thanks for this hint.
> Hyperledger Explorer is working without any issues with v1.1.3.
> Thus I wouldn't merge your commit.
> Are you okay with that?
> It would be great if you could very that the latest commit is working at your machine.
> 
> Best
> Robin

no problem at all @udosson, I will verify the latest changes on my machine and close this issue 👍

opentaps commented:
@udosson What do you think of this?

udosson commented:
@opentaps I would close this pull request. There are no issues to fix and the master branch is up to date with the current version of Hyperledger Explorer.
